data/json/recipes/armor/head.json

```
@@ -2663,6 +2663,27 @@
       { "proficiency": "prof_armorsmithing" }
     ]
   },
+  {
+    "result": "xl_helmet_kettle",
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_HEAD",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "time": "8 h 20 m",
+    "book_learn": [ [ "textbook_armwest", 4 ] ],
+    "using": [ [ "blacksmithing_standard", 6 ], [ "steel_standard", 2 ] ],
```

```suggestion
    "using": [ [ "blacksmithing_standard", 12 ], [ "steel_standard", 3 ] ],
```

Helmet weighs 3.19kg so using only 2kg to craft it doesn't really fly well. ``blacksmithing_standard`` is set for a chunk, one ``steel_standard`` is 4 chunks so 3x4=12. I think we can assume the .19kg is from the fabrics

The regular kettle helmet is actually generating mass in it's recipe, too, so if you have the time and energy I'd ask you to edit it as well. It's fine if you don't want to, though. You're not obligated to fix that.